src/features/ModelPlan/ChangeHistory/util.tsx

```diff
@@ -747,7 +759,11 @@ export const getHeaderText = (change: ChangeRecordType): string => {
       headerText = i18next.t(`changeHistory:planStatusUpdate`);
       break;
     case 'taskListStatusUpdate':
-      if (status === 'In progress') {
+      if (
+        status === 'In progress' &&
+        oldStatus !== 'Complete' &&
```

Comment:
Is there a reason this is `Complete` instead of `Completed`/`COMPLETED`?

Reply:
The statuses come through change history as `Complete` not `Completed`.  Maybe I'm misunderstanding, happy to sync